projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -315,16 +348,15 @@ function grofiles_hovercards_data_html( $author ) {
 	if ( ! $hash ) {
 		return;
 	}
-?>
-	<div class="grofile-hash-map-<?php echo $hash; ?>">
-<?php	foreach ( $data as $key => $value ) : ?>
+	?>
+	<div class="grofile-hash-map-<?php echo esc_html( $hash ); ?>">
```

Comment:
I think you could use `esc_attr()` here, since we're inside an HTML attribute.

```suggestion
	<div class="grofile-hash-map-<?php echo esc_attr( $hash ); ?>">
```